### DISCUSSION TITLE
In linear dimensions, how to find the scientific notation method of dimensioning

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#294] [General]
such as：
![image](https://user-images.githubusercontent.com/59160360/103865232-d4f9fb80-50fe-11eb-8758-a3c15f172d0f.png)

**Comment by mozman** [comment]
All supported DIMENSION features are explained in this tutorial: https://ezdxf.mozman.at/docs/tutorials/linear_dimension.html
